cli/cmd/plugin/standalone-cluster/kapp/kapp.go

```diff
@@ -77,7 +88,7 @@ func New(kubeconfig string) KappManager {
 
 	// setting up restMapper is an expensive operation, so do it here and re-use it
 	// in method invocations
-	groupResources, err := restmapper.GetAPIGroupResources(clientSet.Discovery())
+	groupResources, _ := restmapper.GetAPIGroupResources(clientSet.Discovery())
```

Comment:
Should we check the error here as well?

Reply:
Looks like a number of places in this file we are ignoring errors - I don't have much context on the `kapp` package, so if this is going to be refactored anyways, no worries